packages/skip-link/src/SkipLinkItem.tsx

```diff
@@ -1,15 +1,16 @@
-import { forwardRef, AnchorHTMLAttributes } from 'react';
+import { forwardRef, PropsWithChildren } from 'react';
 import { buttonStyles } from '@ag.ds-next/button';
 import { visuallyHiddenStyles } from '@ag.ds-next/a11y';
 import { mapSpacing } from '@ag.ds-next/core';
 
-export type SkipLinkItemProps = AnchorHTMLAttributes<HTMLAnchorElement>;
+export type SkipLinkItemProps = PropsWithChildren<{ href: string }>;
```

Comment:
Skip links should only ever accept a `href` prop. They do not need to open in new tabs and their content should be self descriptive so need for aria attributes.